src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.html

```diff
@@ -0,0 +1,128 @@
+<div class="Complex-navigator">
```

Comment:
This component has gotten too big and we should split it into multiple ones. Don't worry about it for now, as we may need to use events when we split it on multiple components. I'll take a look at splitting this and then we can discuss it next week.